packages/fhir-converter/src/lib/outputProcessor/resourceMerger.js

```diff
@@ -53,6 +54,68 @@ const concatAndDedup = (target, source) => {
   return destination;
 };
 
+function hasEmptyResourceFilter(item) {
+  const resource = item.resource;
+  if (!resource) return false;
+
+  const keys = Object.keys(resource);
+  const hasMeta = keys.includes("meta");
+  const hasId = keys.includes("id");
+  const hasIdentifier = keys.includes("identifier");
+
+  if ((hasMeta && hasId && keys.length === 3) ||
+      (hasId && keys.length === 2) ||
+      (hasMeta && hasIdentifier && hasId && keys.length === 4)) {
+    return true; // Matches the dead resource criteria, should be filtered out
+  }
+
+  return false; // Does not match the criteria, should not be filtered out
+}
+
+function hasEmptyResourceFilterText(item) {
```

Comment:
Will actually be used in post processing in the follow on PR since there is slightly more complexity here re: removing empty resources and there references